src/python/grpcio_reflection/grpc_reflection/v1alpha/BUILD.bazel

```diff
@@ -28,6 +28,11 @@ py_grpc_library(
     deps = ["reflection_py_pb2"],
 )
 
+py_library(
+    name = "grpc_reflection_test",
```

Comment:
Let's put it into `grpc_reflection`

Reply:
oops. i must have copied this by mistake. `grpc_reflection` below already covers this. removing it.